packages/fxa-settings/src/pages/Signin/SigninRecoveryMethod/index.test.tsx

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import FormChoice from '.';
+import { renderWithLocalizationProvider } from 'fxa-react/lib/test-utils/localizationProvider';
+import { screen, within } from '@testing-library/react';
+import SigninRecoveryMethod from '.';
+
+function render() {
+  renderWithLocalizationProvider(<SigninRecoveryMethod />);
+}
+
+describe('FormChoice', () => {
```

Comment:
Should this be called something else? 'FormChoice' is already used in the FormChoice component?